### ISSUE TITLE
Personal places management

### ISSUE THREAD

**Issue by traktor42** [osmandapp/Osmand#8611]
Home and Work points can be edited only from the route planning panel, but there is no way to clear/delete them. 
It's not possible to edit Home and Work points from "favorities" list. 
"Personal" is a special category which does not allow to switch visibility and colour options. 
"Personal" category is always on top

Proposal 1: 
- Add personal point delete operation from the route planning panel

Proposal 2: 
- Treat "Personal" as usual category in favourites 
- Allow Renaming "Personal" category and alphabetically sort as all other categories
- Allow editing Home and Work points 
Proposal 2 alternative: 
- Don't show "Personal" category in favourites at all

**Comment by vshcherb** [comment]
Points could be deleted as usual - one by one

**Comment by traktor42** [comment]
I can add or edit Home and Work points only from route planning screen.
I can delete Home and Work points only from Favourites screen.
But all operations should be available from the same place. I suggest to add one more control, as on pic

<img width="360" alt="Screenshot_20200304-212537" src="https://user-images.githubusercontent.com/32077668/75911634-e7613400-5e60-11ea-9ab4-f279cccbb57d.png">

**Comment by xerus2000** [comment]
Yes, managing Home is currently a hassle. I struggled for about half an hour until stumbling upon this issue which explains it somewhat accidentally.
I have to go into route navigation, long press on "Home" and then enter an address. If I select a Favorite instead of entering the address, the home point isn't changed.
Just updated to 3.5.5.

I really think they should be a category in "My Places" - what if you commute between more than two places regularly? Or on the other hand, don't have a fixed workplace and want to remove it? Doesn't seem possible within the current system.

**Comment by xerus2000** [comment]
If with the Favorites Screen you mean "My Places", Home and Work points don't show up there for me.

**Comment by pelyg** [comment]
You should add Home or/and Work points from route planning screen. Then they'll be available under **Personal** category in **My Places**

**Comment by xerus2000** [comment]
I added them, but there is no "Personal" category for me in "My Places"...

**Comment by pebogufi** [comment]
@xerus2000 
Don't use outdated program versions to test recent features.

**Comment by jkufner** [comment]
I do not like the "Home" and "Work" at all. These buttons just assume that people have nowhere else to go. What if I don't go to work because I work remotely, but I often travel between three other places? Or what if I go to school instead of work?

I think these buttons should be replaced with something different. I'm thinking about possibility to pin any favourite place. Just like the screen shows the recent places, it could show pinned places. The recent list could have a pin button and the pinned list could have a handle to rearrange the items and to unpin them.

I don't like the use of a category for this purpose, because it requires us to move favorite points to this category from the category to which they belong.

**Comment by xerus2000** [comment]
@pebogufi I just updated it from F-Droid. I thought that was supposed to be the latest version?

**Comment by pebogufi** [comment]
I don't know, it should be. But what about looking on the version number ?
You find it in Help, scroll down ...

**Comment by xerus2000** [comment]
> Just updated to 3.5.5.

I did, and mentioned it already. But apparently I need to update multiple times, or maybe something went wrong in the update.

**Comment by pebogufi** [comment]
Version 3.5.5 is dated nov 20**19**, so, 3.5.5 is quite old.
see 
https://download.osmand.net/releases/
for release version numbers and dates.
The latest release version is 3.8.5 (2020-11-13).
I hope the version numbers in f-droid don't differ.
Currently 3.9.x is at beta state.
